### ISSUE TITLE
Windows WSAEADDRNOTAVAIL error

### ISSUE THREAD

**Issue by NCrashed** [audreyt/network-multicast#12]
[multicastReceiver](https://github.com/audreyt/network-multicast/blob/master/src/Network/Multicast.hsc#L75) throws `WSAEADDRNOTAVAIL` error on Windows platform:
```
bind: failed (Cannot assign requested address (WSAEADDRNOTAVAIL))
```
Platform: Windows 7 x86_64

See: https://github.com/haskell-distributed/distributed-process-simplelocalnet/issues/18

Demo project (particular commit as next commit contains fix that I am going to PR): https://github.com/NCrashed/dp-issue18/tree/242a25cd2072e6b57751f8fb3a50b44e51598cc7

Fix is ready, I will send PR as soon as I finish regression tests in Linux.

**Comment by NCrashed** [comment]
@audreyt the release is great, but `0.1.0` version is also affected by the bug.

**Comment by audreyt** [comment]
Re-released as 0.1.1.

**Comment by NCrashed** [comment]
Thank you!
